src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```

Comment:
not familiar with this, what is it?

Reply:
This comes default with CRA. We can remove it if need be. It helps with web reporting and such